libs/scripts/delayed.js

```diff
@@ -62,7 +62,8 @@ const loadDelayed = ([
     loadJarvisChat(getConfig, getMetadata, loadScript, loadStyle);
     if (getMetadata('interlinks') === 'on') {
       const path = `${getConfig().locale.contentRoot}/keywords.json`;
-      import('../features/interlinks.js').then((mod) => { mod.default(path); resolve(mod); });
+      const language = getConfig().locale.lang ?? getConfig().locale.ietf?.split('-')[0];
```

Comment:
@hparra is making some changes to how we use locales, and it is likely to effect how the config is used here regarding `locale.ietf`. Something to note.

Reply:
Thanks for pointing that out @JasonHowellSlavin I am aware of that change. Hence added both lang and ietf to get the language. So that if lang is not defined for a project yet we can get language from ietf. We can delete ietf from here once it is removed from config.

Reply:
Yes, having the backwards-compatibility like this is the recommended approach.

Reply:
```suggestion
      const { locale } = getConfig();
      const language = locale.lang ?? locale.ietf?.split('-')[0];
```

Reply:
Thanks for the review @narcis-radu . I have incorporated them. Didn't merged your suggestion directly getConfig() was used to get keywords file path also. Hence updated like below
<img width="945" alt="Screen Shot 2023-07-17 at 2 43 01 PM" src="https://github.com/adobecom/milo/assets/69535463/e6bb7047-b321-4ecb-a68b-107180295615">